### PR TITLE
PEP 1: Update language on PEP-Delegate field

### PR DESCRIPTION
**Pull request by willingc**
This PR is a revision of the closed PR #1445.

### CODE REVIEW

**Comment by willingc** [comment]
@brettcannon @warsaw This was a PEP change that our earlier Steering Council decided would be helpful. Unfortunately the earlier PR got stale. Let's approve and merge this one :smile:

File: pep-0001.txt [hunk 1]
```diff
@@ -281,14 +281,24 @@ notifying the PEP author(s) and giving them a chance to make revisions.
...
+authority to approve (or reject) that PEP. 
+
+The terms, PEP-Delegate (preferred term under the Steering Council governance 
+model) and BDFL-Delegate, may be used interchangeably in discussion.
+The term "PEP-Delegate" will be used under the Steering
+Council governance model. The PEP's designated decision maker,
+the "PEP-Delegate" with the Steering Council's support, is
+recorded in the "BDFL-Delegate" field in the PEP's header. For consistency with
+PEPs written prior to the Steering Council's governance model, the field name,
+"BDFL-Delegate" will still be kept but the field means "PEP-Delegate".
```
Comment (JelleZijlstra) [inline pep-0001.txt:294]: We actually ended up changing this, and new PEPs use the PEP-Delegate field:
```
pep-0639.rst:PEP-Delegate: Paul Moore <p.f.moore at gmail.com>
pep-0641.rst:PEP-Delegate: Pablo Galindo <pablogsal@python.org>
pep-0656.rst:PEP-Delegate: Paul Moore <p.f.moore@gmail.com>
pep-0658.rst:PEP-Delegate: Donald Stufft <donald@stufft.io>
pep-0665.rst:PEP-Delegate: Paul Moore <p.f.moore@gmail.com>
```

Comment (AA-Turner) [reply pep-0001.txt:294]: Should we note that the PEP-Delegate field is used with the advent of the SC model, but BDFL-Delegate is a legacy alias? (Just ensuring we explain what both mean, as I think it is unlikely we will mass update all old PEPs to use PEP-Delegate.)

File: pep-0001.txt [hunk 2]
```diff
@@ -281,14 +281,24 @@ notifying the PEP author(s) and giving them a chance to make revisions.
...
 experienced to make the final decision on that PEP may offer to serve as
-the PEP-Delegate for that PEP, and they will then have the authority to approve
-(or reject) that PEP. Individuals taking on this responsibility are free to seek
+the PEP-Delegate (BDFL-Delegate) for that PEP, and they will then have the
+authority to approve (or reject) that PEP. 
+
+The terms, PEP-Delegate (preferred term under the Steering Council governance 
+model) and BDFL-Delegate, may be used interchangeably in discussion.
+The term "PEP-Delegate" will be used under the Steering
+Council governance model. The PEP's designated decision maker,
```
Comment (AA-Turner) [inline pep-0001.txt:290]: The second sentence is a stronger phrasing than the first ("will be used" vs "preferred term"). I'd suggest keeing the latter, and moving the explanation that the terms mean the same thing to the end of the paragraph.

File: pep-0001.txt [hunk 3]
```diff
@@ -281,14 +281,24 @@ notifying the PEP author(s) and giving them a chance to make revisions.
 The final authority for PEP approval is the Steering Council. However, whenever
 a new PEP is put forward, any core developer that believes they are suitably
 experienced to make the final decision on that PEP may offer to serve as
-the PEP-Delegate for that PEP, and they will then have the authority to approve
-(or reject) that PEP. Individuals taking on this responsibility are free to seek
+the PEP-Delegate (BDFL-Delegate) for that PEP, and they will then have the
```
Comment (AA-Turner) [inline pep-0001.txt:284]:
```suggestion
the PEP-Delegate for that PEP, and they will then have the
```

**Review by AA-Turner — CHANGES_REQUESTED** [review]

File: pep-0001.txt (hunk 1 above)
Comment (willingc) [reply pep-0001.txt:294]: Good suggestion. Nope, we don't want to update the old PEPs.

File: pep-0001.txt [hunk 4]
```diff
@@ -281,14 +281,24 @@ notifying the PEP author(s) and giving them a chance to make revisions.
 The final authority for PEP approval is the Steering Council. However, whenever
 a new PEP is put forward, any core developer that believes they are suitably
 experienced to make the final decision on that PEP may offer to serve as
-the PEP-Delegate for that PEP, and they will then have the authority to approve
-(or reject) that PEP. Individuals taking on this responsibility are free to seek
+the PEP-Delegate for that PEP, and they will then have the
+authority to approve (or reject) that PEP. 
+
+The term "PEP-Delegate" will be used under the Steering
```
Comment (AA-Turner) [inline pep-0001.txt:287]:
```suggestion
The term "PEP-Delegate" is used under the Steering
```

File: pep-0001.txt [hunk 5]
```diff
@@ -281,14 +281,24 @@ notifying the PEP author(s) and giving them a chance to make revisions.
...
+Council governance model. The PEP's designated decision maker,
+the "PEP-Delegate" with the Steering Council's support, is
+recorded in the "PEP-Delegate" field in the PEP's header. For
+PEPs written prior to the Steering Council's governance model, the field name,
```
Comment (AA-Turner) [inline pep-0001.txt:291]:
```suggestion
PEPs written prior to the Steering Council's governance model, the field name
```

File: pep-0001.txt [hunk 6]
```diff
@@ -281,14 +281,24 @@ notifying the PEP author(s) and giving them a chance to make revisions.
...
+"BDFL-Delegate" will still be kept.
+The terms, PEP-Delegate (preferred term under the Steering Council governance 
+model) and BDFL-Delegate, may be used interchangeably in discussion.
```
Comment (AA-Turner) [inline pep-0001.txt:294]:
```suggestion
The terms PEP-Delegate and BDFL-Delegate may be used interchangeably in 
discussion. PEP-Delegate is the preferred term under the Steering Council 
governance model.
```

**Review by AA-Turner — APPROVED** [review]
Looks good

Three very minor suggestions (tense and commas), feel free to ignore though

**Review by brettcannon — APPROVED** [review]

File: pep-0001.txt (hunk 1 above)
Comment (CAM-Gerlach) [reply pep-0001.txt]: > Should we note that the PEP-Delegate field is used with the advent of the SC model, but BDFL-Delegate is a legacy alias? (Just ensuring we explain what both mean, as I think it is unlikely we will mass update all old PEPs to use PEP-Delegate.)

:+1: That's also the approach taken in other governance-related PEPs that mention this.

**Review by warsaw — APPROVED** [review]
Thanks @willingc and @AA-Turner !

File: pep-0001.txt (hunk 6 above)
Comment (CAM-Gerlach) [reply pep-0001.txt]: I might restructure this a bit to mention PEP-Delegate first and BDFL-Delegate as a legacy alias that should be understood to mean the same thing, but it's not terribly important.

File: pep-0001.txt [hunk 7]
```diff
@@ -281,14 +281,25 @@ notifying the PEP author(s) and giving them a chance to make revisions.
...
 experienced to make the final decision on that PEP may offer to serve as
-the PEP-Delegate for that PEP, and they will then have the authority to approve
-(or reject) that PEP. Individuals taking on this responsibility are free to seek
+the PEP-Delegate for that PEP, and they will then have the
+authority to approve (or reject) that PEP. 
+
+The term "PEP-Delegate" is used under the Steering
+Council governance model. The PEP's designated decision maker,
+the "PEP-Delegate" with the Steering Council's support, is
+recorded in the "PEP-Delegate" field in the PEP's header. For
```
Comment (CAM-Gerlach) [inline pep-0001.txt:290]:
```suggestion
The term "PEP-Delegate" is used under the Steering Council governance model
for the PEP's designated decision maker, as accepted by the Steering Council,
and recorded in the "PEP-Delegate" field in the PEP's header. For
```

I found the wording here a little choppy, redundant and hard to parse, and suggest a bit of a restructuring to improve this.

File: pep-0001.txt [hunk 8]
```diff
@@ -281,14 +281,25 @@ notifying the PEP author(s) and giving them a chance to make revisions.
...
+PEPs written prior to the Steering Council's governance model, the field name
```
Comment (CAM-Gerlach) [inline pep-0001.txt:291]:
```suggestion
PEPs written prior to the Steering Council governance model, the field name
```

Nitpick, but the possessive implies that the Steering Council owns/posses the governance model, as opposed to itself being the name of the governance model

File: pep-0001.txt [hunk 9]
```diff
@@ -281,14 +281,25 @@ notifying the PEP author(s) and giving them a chance to make revisions.
...
+"BDFL-Delegate" will still be kept.
+The terms PEP-Delegate and BDFL-Delegate may be used interchangeably in 
+discussion. PEP-Delegate is the preferred term under the Steering Council 
+governance model.
```
Comment (CAM-Gerlach) [inline pep-0001.txt:295]:
```suggestion
The term "BDFL-Delegate" is a legacy alias for PEP-Delegate, stemming from
the time when Python was led by a BDFL, but otherwise has the same meaning.
```

This emphasizes that BDFL-Delegate is a legacy alias and explains its origin, while being simpler and more direct and with less choppiness.

File: pep-0001.txt [hunk 10]
```diff
@@ -281,14 +281,25 @@ notifying the PEP author(s) and giving them a chance to make revisions.
...
+
+Individuals offering to serve as PEP-Delegate should notify the Steering
+Council, PEP authors and PEP sponsor of their intent to self-nominate.
+Individuals taking on this responsibility are free to seek
```
Comment (CAM-Gerlach) [inline pep-0001.txt:299]:
```suggestion
Those taking on this responsibility are free to seek
```

Avoid repetition

**Review by CAM-Gerlach — COMMENTED** [review]
Seems like the PR was merged before I submitted this review, but I can open another PR to refine the wording

**Comment by warsaw** [comment]
I already merged this PR.  I'd rather not revert.  Can you create a new one with any follow up changes?

**Comment by CAM-Gerlach** [comment]
Yeah sure, of course—I was going to go ahead with another PR; just wanted to wait to see if anyone objected. Thanks!
